tests/utils_tests/test_html.py

```diff
@@ -8,6 +8,9 @@
 from django.utils import html, safestring, six
 from django.utils._os import upath
 from django.utils.encoding import force_text
+from django.utils.functional import lazy
+
+lazystr = lazy(force_text, six.text_type)
```

Comment:
Maybe it's worth adding this as a helper in utils.functional (since it seems to be a common case like keep_lazy_text) rather than repeating in a bunch of tests?